src/verifiers/documentStatus/didSignedDocumentStatus/didSignedDocumentStatus.ts

```diff
@@ -20,15 +23,134 @@ const skip: VerifierType["skip"] = async () => {
 };
 
 const test: VerifierType["test"] = (_document) => {
-  return true;
+  const document = _document as any; // TODO Casting to any first to prevent change at the OA level
+  if (document.proof && document.proof.some((proof: any) => proof.type === "DidGenericSignature")) return true;
```

Comment:
![image](https://user-images.githubusercontent.com/7406870/91926064-fd18be80-ed08-11ea-942b-ecde69635083.png)

That is the suite of signature. In this case we are using DID as a proxy for the actual key being used, so I'm not sure if the key type itself should be used in this case...

Reply:
Hmm there is two ways to approach this:

1. We create a new class to show that it is proxying through DID. This way it wont be confused with the existing jws method. Another compliant verifier will not attempt to process this document the way they would have with the `Secp256k1VerificationKey2018` type and is expecting a `jws` key. 

2. We stick to the key type directly, we do not invent a new type. The problem could be that existing compliant verifier will attempt to process it and fail since it was expecting a jws.

Reply:
Existing specs: https://w3c-ccg.github.io/ld-cryptosuite-registry

Reply:
@rjchow suggested that we could use the key we were going to use in V3 as shown here https://github.com/Open-Attestation/open-attestation/blob/master/src/schema/3.0/w3c.test.ts#L41

Reply:
Have moved it to use `OpenAttestationSignature2018`